python/paddle/fluid/tests/unittests/dygraph_to_static/test_dict.py

```diff
@@ -203,5 +203,43 @@ def _set_test_func(self):
         self.dygraph_func = test_dic_pop_2
 
 
+class NetWithDictPop(paddle.nn.Layer):
+    def __init__(self):
+        super(NetWithDictPop, self).__init__()
+
+    @to_static
+    def forward(self, x, **kwargs):
+        x = paddle.to_tensor(x)
+        y = kwargs.pop('y', None)
+        if y is True:
```

Comment:
Tiny issue:

`if y is True` -> `if y` is more common. Since this issue is very tiny, I approved this PR. You can change later.